thanks/cli.py

```diff
@@ -1,20 +1,48 @@
 # -*- coding: utf-8 -*-
 
 """Console script for thanks."""
+from __future__ import absolute_import
+
+import logging
 
 import click
-from .thanks import Thanks
+from thanks.thanks import Thanks
+
+logging.basicConfig(level=logging.CRITICAL)
+logger = logging.getLogger("thanks")
 
 
 @click.command()
-@click.argument('requirements', default='requirements.txt', type=click.Path(exists=True))
-@click.option('--debug', is_flag=True, help='Set debug mode')
-def main(requirements, debug):
-    """Console script for thanks."""
-    with open(requirements, 'r') as fh:
-        requirements = fh.read().strip().split()
-    Thanks(debug=debug).find_project_details(requirements)
-    return 0
+@click.argument("package_name",
+                nargs=-1)
+@click.option("--requirements", "-r",
+              multiple=True,
+              type=click.File("r"))
+@click.option("--pipfile", "-p",
+              multiple=True,
+              type=click.File("r"))
+# @click.option("--setuppy", "-s",
```

Comment:
I think it's good to support these in the future, but would rather not have commented out code lying around. If we want to add a `# TODO: Support more types` I'm fine with that.